src/apiCalls.js

```diff
@@ -2,7 +2,7 @@ export const fetchApiData = (type) => {
   return fetch(`http://localhost:3001/api/v1/${type}`)
     .then(response => {
       if (!response.ok) {
```

Comment:
I think both of us could DRY up these conditionals by creating a checkForError function like [here](https://github.com/zachjjohns/travel-tracker/blob/main/src/API.js)